recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - python >=3.6
     - setuptools
     - conda >=4.2
-    - conda-build >=3.18.3
+    - conda-build >=3.28.2
```

Comment:
Should be compatible with older versions too -- but I'm also fine to up this.

Reply:
yep sorry - fixing now.